charts/intel-resource-drivers/README.md

```diff
@@ -33,10 +40,14 @@ You may also run `helm show values` on this chart's dependencies for additional
 
 | Key | Type | Default |
 |-----|------|---------|
-| image.repository | string | `intel` |
-| image.name | string | `"intel-resource-drivers"` |
-| image.pullPolicy | string | `"IfNotPresent"` |
-| image.tag | string | `"v0.5.1"` |
+| gpu.image.repository | string | `intel` |
+| gpu.image.name | string | `"intel-gpu-resource-driver"` |
+| gpu.image.pullPolicy | string | `"IfNotPresent"` |
+| gpu.image.tag | string | `"v0.5.1"` |
+| gaudi.image.repository | string | `intel` |
```

Comment:
Just FYI - image name is repository and tag: https://docs.docker.com/guides/docker-concepts/building-images/build-tag-and-publish-an-image/#tagging-images

- `intel` is the name of the organization / namespace in Docker Hub
- `intel/intel-gpu-resource-driver` is the name of the repository
- `v0.5.1` is the tag
- `intel/intel-gpu-resource-driver:v0.5.1` is full image name

This can be fixed later, I don't mind.

Reply:
Perhaps add most important values here - the ones that enable installation of the driver ?